compiler/rustc_const_eval/src/const_eval/eval_queries.rs

```diff
@@ -299,19 +294,17 @@ pub fn eval_to_allocation_raw_provider<'tcx>(
     let cid = key.value;
     let def = cid.instance.def.with_opt_param();
     let is_static = tcx.is_static(def.did);
+    let mut interpreter = CompileTimeInterpreter::new();
+    // Statics (and promoteds inside statics) may access other statics, because unlike consts
+    // they do not have to behave "as if" they were evaluated at runtime.
+    if tcx.is_static(def.did) {
```

Comment:
```suggestion
    let mut interpreter = CompileTimeInterpreter::new();
    // Statics (and promoteds inside statics) may access other statics, because unlike consts
    // they do not have to behave "as if" they were evaluated at runtime.
    if tcx.is_static(def.did) {
```

Reply:
This suggestion got lost?